www/js/controllers.js

```diff
@@ -73,12 +86,14 @@ angular.module('wpApp.controllers', [])
 
     DataLoader.get( siteApi ).then(function(response) {
 
-        var siteID = $rootScope.increment();
-        var site = { id: siteID, title: response.data.name, description: response.data.description, url: siteURL, username: u.username, password: u.password };
-        $scope.sites.push( site );
+        var site = { _id: new Date().toISOString(), title: response.data.name, description: response.data.description, url: siteURL, username: u.username, password: u.password };
```

Comment:
It was recommended in the PouchDB documentation http://pouchdb.com/getting-started.html

If we want to use something else, that's totally fine. I just didn't want to get it from localStorage anymore just in case that got wiped.